README.md

```diff
@@ -10,6 +10,18 @@ Opinionated stacks of ready-to-run Jupyter applications in Docker.
 If you're familiar with Docker, have it configured, and know exactly what you'd like to run, this one-liner should work in most cases:
 
 ```
+# Foreground mode:
+# -ti: pseudo-TTY+STDIN open.
+# -rm: remove the container on exit.
+# -p: publish port to the host
+docker run -ti --rm -p 8888:8888 jupyter/<your desired stack>:<git-sha-tag>
+
+# And mount current directory into 'work' directory in image
```

Comment:
Maybe expand this comment to say why you'd pick this form (retain your work across containers) vs the first (throwaway instance).

Reply:
Also, something probably needs to be said about permissions on `$PWD` so that the default 1000:100 (jovyan:user) in the container can write to it.

Reply:
Hi @parente.

When you mount a volume, my test showed you don't need to change permissions.
Permission change is only for internal folders in image, for example after a COPY command. Is this right?

Reply:
It appears to depend on either the version of docker or the OS. The latest Docker for Mac seems to allow `jovyan` read-write access to the volume when mounted over `/home/jovyan`. The version of Docker running on Travis CI for Linux seems to respect the ownership and permissions of the host directory when mounted as volume, and disallows write access by `jovyan` by default (ref: integration test failure https://travis-ci.org/jupyter/docker-stacks/builds/310366105).